src/rp2_common/pico_sd_card/sd_card.c

```diff
@@ -828,6 +828,10 @@ bool sd_scatter_read_complete(int *status) {
         }
         check_crc_count = 0;
     }
+    else if (blocks_complete)
+    {
```

Comment:
brace-error :laughing:

Reply:
I do normally try to fit in with surrounding code's style, but I'm very much used to braces being on their own line!  Will try to remember.